tests/test_validator_integration.py

```
@@ -0,0 +1,970 @@
+class TestRemoveUnusedColumns:
+    """GLiNER uses custom data collators.  HF TrainingArguments defaults
+    remove_unused_columns=True which strips columns the collator needs.
+    Fixed: create_training_args now defaults to False, and training_cli forwards it."""
+
+    def test_default_remove_unused_columns_is_true(self):
+        """The HF default for remove_unused_columns is True."""
+        import tempfile
+
+        pytest.importorskip("torch", reason="requires torch")
+        from gliner.training.trainer import TrainingArguments
+        import tempfile
```

Ruff (0.15.1)

[error] Redefinition of unused `tempfile` from line 554: `tempfile` redefined here

Remove definition: `tempfile`

(F811)